packages/stager/example/lib/main.dart

```diff
@@ -0,0 +1,33 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
This file is currently your example tab on pub.dev, which is pretty confusing. You should strongly consider adding an example.md per https://dart.dev/tools/pub/package-layout#examples.